### ISSUE TITLE
Background color is always transparent when we export a chart

### ISSUE THREAD

**Issue by domoritz** [vega/vega#543]
```json
{
  "width": 400,
  "height": 200,
  "padding": {"top": 10, "left": 30, "bottom": 30, "right": 10},
  "background": "red",
  "data": [
    {
      "name": "table",
      "values": [
        {"x": 1,  "y": 28}, {"x": 2,  "y": 55},
        {"x": 3,  "y": 43}, {"x": 4,  "y": 91},
        {"x": 5,  "y": 81}, {"x": 6,  "y": 53},
        {"x": 7,  "y": 19}, {"x": 8,  "y": 87}
      ]
    }
  ],
  "scales": [
    {
      "name": "x",
      "type": "ordinal",
      "range": "width",
      "domain": {"data": "table", "field": "x"}
    },
    {
      "name": "y",
      "type": "linear",
      "range": "height",
      "domain": {"data": "table", "field": "y"},
      "nice": true
    }
  ],
  "axes": [
    {"type": "x", "scale": "x"},
    {"type": "y", "scale": "y"}
  ],
  "marks": [
    {
      "type": "rect",
      "from": {"data": "table"},
      "properties": {
        "enter": {
          "x": {"scale": "x", "field": "x"},
          "width": {"scale": "x", "band": true, "offset": -1},
          "y": {"scale": "y", "field": "y"},
          "y2": {"scale": "y", "value": 0}
        },
        "update": {
          "fill": {"value": "steelblue"}
        },
        "hover": {
          "fill": {"value": "red"}
        }
      }
    }
  ]
}
```

In the editor:
![screen shot 2016-03-01 at 11 01 14](https://cloud.githubusercontent.com/assets/589034/13438222/f6292e1c-df9c-11e5-9cc3-5f7245ce141c.png)

The generated PNG:
![screen shot 2016-03-01 at 11 01 22](https://cloud.githubusercontent.com/assets/589034/13438238/04c4ef06-df9d-11e5-8bf1-fce94129f28f.png)

**Comment by domoritz** [comment]
Noe a vega bug.

**Comment by domoritz** [comment]
The bug is in `view.toImageURL`.

**Comment by arvind** [comment]
Aha so it _is_ a Vega bug! Good catch!

**Comment by arvind** [comment]
Fixed with 39dff1f. Thanks @domoritz and @hrbrmstr.
